src/ContainerInstance/docs/New-AzContainerGroup.md

```diff
@@ -29,7 +29,7 @@ New-AzContainerGroup -Name <String> -ResourceGroupName <String> -Container <ICon
 ```
 
 ## DESCRIPTION
-Create or update container groups with specified configurations. For property Volume, we support mounting Azure File share as a volume or specifying git repository as volume directory. The empty directory volume and secret volume are not supported yet.
+Create or update container groups with specified configurations.
```

Comment:
The second sentence seems to be added manually, so after regenerating it's gone.
We should add it back through directive or customization.

Reply:
I deleted the sentence intentionally. Because we will support empty directory volume and secret volume after the PR merged.